packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -96,6 +97,18 @@ export class TableRow<
         this.emitToggleEvent('row-action-menu-toggle', event.detail, column);
     }
 
+    public closeOpenActionMenus(): void {
```

Comment:
Could you add a unit test for this function? If it's only possible/simple to test it indirectly then that's not a big deal, but having a test for it in table-row.spec.ts would be best.

Reply:
I don't think there's a simple/ straightforward way to unit test it, since it's interacting with the action menus and that code goes through multiple layers (cell / row / column / table). It's tested indirectly via the `uses virtualization and closes open action menus when a scroll happens` test in `table.spec.ts`.